ext/oauth-client/Civi/Api4/Action/OAuthClient/AbstractGrantAction.php

```diff
@@ -47,10 +47,6 @@ abstract class AbstractGrantAction extends \Civi\Api4\Generic\AbstractBatchActio
    */
   private $clientDef = NULL;
 
-  public function __construct($entityName, $actionName) {
-    parent::__construct($entityName, $actionName, ['*']);
```

Comment:
does the parent ignore the last param?

Reply:
Yes it's unused.

Reply:
Looks like it does!
![image](https://github.com/civicrm/civicrm-core/assets/336308/9d985921-101d-4837-bcd3-580e8e65c0e6)